index.js

```diff
@@ -31,9 +31,10 @@ function sseMiddleware(__, res, next) {
     if (id) {
       res.write(`id: ${id}\n`);
     }
-
+    if (evt) {
```

Comment:
Since the event name is optional, we should also change the order of arguments to `res.sse = (json, evt, id) => {`. Can you please update whole the code base (usages of `res.sse(...)`) according this suggestion?